tensorflow/tools/compatibility/tf_upgrade_v2_main.py

```diff
@@ -22,15 +22,17 @@
 
 from tensorflow.tools.compatibility import ast_edits
 from tensorflow.tools.compatibility import tf_upgrade_v2
+from tensorflow.tools.compatibility import ipynb
 
 
 def main():
   parser = argparse.ArgumentParser(
       formatter_class=argparse.RawDescriptionHelpFormatter,
-      description="""Convert a TensorFlow Python file to 2.0
+      description="""Convert a TensorFlow Python file from 1.* to 2.0
```

Comment:
I prefer 1.x

Reply:
funny, since I started as `1.x`, but in a header, it was already written as `1.*`